scripts/docker/submarine/1.0.0/zeppelin-cpu-submarine-interpreter/Dockerfile

```diff
@@ -0,0 +1,20 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+FROM zeppelin-cpu-tensorflow_1.13.0-hadoop_3.1.2:1.0.0
```

Comment:
Where is it ? I could not find it in dockerhub

Reply:
Sorry, I see it below of this PR

Reply:
`Zeppelin-cpu-tensorflow_1.13.0-hadoop_3.1.2` Is the basic image that we need to provide,
Push to dockerhub for users to use, So now there is no dockerhub inside.
The docker images for `zeppelin-cpu-tensorflow_1.13.0-hadoop_3.1.2` are in `scripts/docker/submarine/1.0.0/zeppelin-cpu-tensorflow_1.13.1-hadoop_3.1.2/Dockerfile`.
Regarding the name of the image, felixcheung also made some comments.
I want to do this:
#### Base Docker Image:(Used to run the tensorflow algorithm in yarn)
```
zeppelin hadoop submarine cpu base image name: zeppelin-submarine-cpu:1.0.0, Comment: tensorflow_1.13.0-cpu & hadoop_3.1.2
zeppelin hadoop submarine gpu base image name: zeppelin-submarine-gpu:1.0.0, Comment: tensorflow_1.13.0-gpu & hadoop_3.1.2
```

#### Interpreter Docker Image:(Used to run the submarine interpreter in yarn)
```
zeppelin hadoop submarine Interpreter cpu image name: zeppelin-submarine-Interpreter-cpu:1.0.0, Comment: tensorflow_1.13.0-cpu & hadoop_3.1.2
zeppelin hadoop submarine Interpreter gpu image name: zeppelin-submarine-Interpreter-gpu:1.0.0, Comment: tensorflow_1.13.0-gpu & hadoop_3.1.2
```

#### Runtime Mount zeppelin lib
There is no need to pre-install zeppelin in docker image.
When zeppelin launches the submarine interpreter via `LaunchOnYarn`,
Will mount the zeppelin submarine lib jar into the docker container.
This very mode version of the update also reduces the size of the image